driver-core/src/test/java/com/datastax/driver/core/utils/CassandraVersion.java

```diff
@@ -34,7 +34,12 @@
     /**
      * @return The minor version required to execute this test, i.e. "0"
      */
-    int minor() default 0;
+    int minor() default -1;
+
+    /**
+     * @return Whether or not the exact major version is exactly required to run this test.
+     */
+    boolean[] exactRequired() default {false};
```

Comment:
`@CCMConfig` is complex in that it needs to distinguish `true`, `false` and `unset`, and so it uses boolean arrays = `{true}`, `{false}` and `{}`. But here I think you should only need:

```java
boolean exactRequired() default false;
```